bench/locli/src/Cardano/Util.hs

```diff
@@ -32,16 +31,20 @@ import Cardano.Prelude
 
 #if __GLASGOW_HASKELL__ < 902
 -- This is a GHC module ...
-import Util
-#else
+import Util                      hiding (fst3, snd3)
+#elif __GLASGOW_HASKELL__ < 906
 -- that moved for the ghc-9.2 release.
 import GHC.Utils.Misc                   as Util
-#endif
                                  hiding (fst3, snd3, third3, uncurry3, firstM, secondM)
+#else
+-- that moved again for the ghc-9.6 release.
+-- Taking an internal module of GHC and re-exporting it is an incredibly dumb idea.
```

Comment:
```suggestion
-- Taking an internal module of GHC and re-exporting it exceptionally fragile; we should
-- work hard to prevent these kinds of dependencies. 
```

The import should have a proper comment as to _why_ we import GHC machinery here. I'm not sure who'd be qualified to write such a comment?